lib/src/utf.dart

```diff
@@ -0,0 +1,252 @@
+// Large portions of this code where taken from https://github.com/dart-lang/utf
+
+import "dart:collection";
+
+const int unicodeReplacementCharacterCodepoint = 0xfffd;
```

Comment:
fyi, I didn't review this file.

Reply:
No worries – it's just copy-paste + a little renaming